.github/workflows/publish.yaml

```diff
@@ -96,7 +95,7 @@ jobs:
 
   build_and_push:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Does this stay on an LTS? Do you care?

Reply:
No, I don't really care what which version it is since it's just building the docker image.  I could imagine a breaking change made in some future docker upgrade, but hopefully I'd be aware of something like that in advance.